### ISSUE TITLE
[Client] Allow to create two client instances

### ISSUE THREAD

**Issue by florianriche** [pyronear/pyro-api#88]
The way a client is instanced right now is buggy (guilty here) because everytime we create a client instance, we alter the **Class** variable routes.
As a consequence, if we want to create two client instances (for whatever reason), the second instance will have incorrect routes as it will use the already altered class variable.

```python

 class Client:
 
   routes = {"token": "/login/access-token",
               ...
              }

    def __init__(self, api_url, credentials_login, credentials_password):
        self.api = api_url
        self._add_api_url_to_routes()
        ...
    def _add_api_url_to_routes(self):
        for k, v in self.routes.items():
            self.routes[k] = urljoin(self.api, v)
     ...
```

Suggested solutions:
* change  the current routes class variable to "_routes" and create a deepcopy self.routes in the constructor before calling _add_api_url_to_routes.
* Refactor the whole class construction.

**Comment by frgfm** [comment]
Thanks for reporting that, I actually tackled it in #74 changing the way the `routes` attribute gets created. That solves the problem on my end, let me know if it doesn't on yours!

**Comment by frgfm** [comment]
@florianriche could you tell me how to reproduce the bug?
If it was indeed fixed by #74 as suggested earlier, we can close this issue :ok_hand:

**Comment by florianriche** [comment]
Solved by #74

**Comment by florianriche** [comment]
@frgfm I close it at samedi tome of your comment !

Just noticed the big when in a terminal trying to  test the client,  id you créate a first api-client and then a second, the second wont ne able to retrieve a token. I think it is solved know.
